billing.py

```diff
@@ -5,32 +5,49 @@
 logger = logging.getLogger(__name__)
 
 BIGQUERY_CLIENT_PROJECT = CONFIG['billing']['bigquery_client_project'].get()
-COST_VIEW_FULL_NAME = CONFIG['billing']['cost_view_full_name'].get()
+BILLING_TABLE_FULL_NAME = CONFIG['billing']['billing_export_table_name'].get()
+COST_WINDOW_DAYS = CONFIG['filters']['age_maximum_days'].get(int)
 
 def query_billing_info():
     client = bigquery.Client(project=BIGQUERY_CLIENT_PROJECT)
-    query_job = client.query("""
+
+    if COST_WINDOW_DAYS and COST_WINDOW_DAYS > 0:
```

Comment:
NIT: 
In Python, if you run an if statement with a numeric variable, it will accept the condition as true if X > 0 and X < 0. So you can simplify this block. 

```suggestion
    if COST_WINDOW_DAYS:
```